query-coordinator/src/main/scala/com/socrata/querycoordinator/fusion/CompoundTypeFuser.scala

```diff
@@ -105,11 +105,15 @@ class CompoundTypeFuser(fuseBase: Map[String, String]) extends SoQLRewrite with
    * Columns involved are prefixed during ast rewrite and removed after analysis to avoid column name conflicts.
    */
   def postAnalyze(analyses: Seq[SoQLAnalysis[ColumnName, SoQLType]]): Seq[SoQLAnalysis[ColumnName, SoQLType]] = {
+    val lastAnalysis = analyses.last
     analyses.map { a =>
-      val selection = a.selection map { case (cn, expr) =>
-        (ColumnName(cn.name.replaceFirst(ColumnPrefix, "")) -> expr)
+      if (a.ne(lastAnalysis)) { a }
```

Comment:
similarly, `analyses.updated(analyes.size - 1, ...)` is likely more straightforward.